tests/bluetooth/controller/ctrl_ull_dletime/src/dle_2m.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2018-2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define FEATURES_TO_TEST 4
+#define OCTETS_TO_TEST 5
+
+static u16_t default_octets[] = {27, 60, 128, 200, 251};
```

Comment:
IIRC the coding style requires space after `{` and before `}` - at least that's what we try to follow within all Bluetooth code, so please try to be consistent with it. I won't comment on other places in this patch, but there are lots of them.

Reply:
Ok, I'll follow what we try to do; not much work either with sed